tinyms/serving/server/server.py

```diff
@@ -90,31 +99,33 @@ def shutdown(self):
         Shutdown the flask server.
         """
 
+        if os.path.exists('temp.json'):
+            os.remove('temp.json')
         if self.system_name == "windows":
             server_res = subprocess.getoutput("netstat -ano | findstr 5000")
             server_pid = None
             for line in server_res.split("\n"):
                 temp = [i for i in line.split(' ') if i != '']
                 if len(temp) > 4:
-                    if temp[1] == '127.0.0.1:5000' and temp[3] == 'LISTENING':
+                    if temp[1] == f'{self.host}:{self.port}' and temp[3] == 'LISTENING':
                         server_pid = temp[4]
                         continue
             if server_pid:
                 os.system("taskkill /t /f /pid %s" % server_pid)
         else:
-            server_pid = subprocess.getoutput("netstat -anp | grep 5000 | awk '{printf $7}' | cut -d/ -f1")
+            server_pid = subprocess.getoutput("netstat -anp | grep %s | awk '{printf $7}' | cut -d/ -f1" % self.port)
             subprocess.run("kill -9 " + str(server_pid), shell=True)
         return 'Server shutting down...'
 
     def windows_run_server(self):
-        cmd = 'python -c "from tinyms.serving import run_flask; run_flask()"'
+        cmd = f"""python -c "from tinyms.serving import run_flask; run_flask('{self.host}', {self.port})"""""
```

Comment:
I wonder is it ok to remove `''` for {self.host}.

Reply:
They are now in a more suitable position